simpeg/electromagnetics/natural_source/__init__.py

```diff
@@ -23,8 +23,11 @@
 .. autosummary::
   :toctree: generated/
 
+  receivers.PointImpedance
   receivers.PointNaturalSource
   receivers.Point3DTipper
+  receivers.Point3DAdmittance
+  receivers.Point3DMobileMT
```

Comment:
A thought on naming... for the MobileMT and elsewhere where we now have different base station and measurement locations, we might want to move away from `Point`